packages/nextjs/src/config/templates/routeHandlerWrapperTemplate.ts

```diff
@@ -58,16 +58,20 @@ function wrapHandler<T>(handler: T, method: 'GET' | 'POST' | 'PUT' | 'PATCH' | '
   });
 }
 
+// Re-export anything exported by the page module we're wrapping. When processing this code, Rollup is smart enough to
+// not include anything whose name matchs something we've explicitly exported above.
```

Comment:
Just a bystander reading this but maybe you can remove the " above" bit at the end since the comment has moved.

Reply:
Ah yea. Thanks <3